SETUP.md

```diff
@@ -0,0 +1,39 @@
+# Nylas Use Case Samples
+
+**⚠️ Warning: This repo is not meant for use in production and stability is not fully guaranteed.**
+
+Explore working front-end and back-end sample code using an integration with Nylas.
+
+The sample application redirects to a prebuilt authentication page hosted on Nylas, then allows that authenticated user to interact with Nylas API from a web client.
+
+## ⚙️ Environment Setup
+
+Let’s check that our environment is set up to use the [Nylas Node SDK](https://github.com/nylas/nylas-nodejs). Check the Node version in your terminal:
+
+```bash
+$ node -v
+v18.0.0
+```
+
+If you don’t see a version returned, you may not have Node installed. Try the following steps:
+
+1. Visit [nodejs.org](https://nodejs.org/en/) to set up Node on your machine
+2. _Recommended_: If you happen to use or require multiple versions of Node, consider using [nvm](https://github.com/nvm-sh/nvm)
```

Comment:
Do you think it might be confusing for a beginner to have this extra information?

Reply:
Removing!